### PR TITLE
(FEV-1351) Navigation plugin documentation update

### PR DESCRIPTION
**Pull request by MishelYeghiazaryan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,23 +98,94 @@ This project is licensed under the AGPL-3.0 License - see the [LICENSE.md](LICEN
...
+>
+> ##### Default: `false`
+>
+> ##### Description: force to use chapters thumbnails.
+>
+
+##
+
+> ### config.expandMode
+>
```
Comment (amirch1) [inline README.md:147]: @MishelYeghiazaryan I think that the expandMode available configuration strings are 'alongside', 'hidden', 'over'. You can verify with @semarche-kaltura

**Review by amirch1 — CHANGES_REQUESTED** [review]

**Comment by amirch1** [comment]
@MishelYeghiazaryan can you close this PR and open a new PR with only the README.md file change? in this PR there are additional changed files including the client library. FYI @semarche-kaltura

**Comment by semarche-kaltura** [comment]
let's changes to few PRs:
1 PR: update for README file;
2 PR: add absent files for eslint and editor config;

File: README.md (hunk 1 above)
Comment (semarche-kaltura) [reply README.md:147]: enums are:
`
{
  ALONGSIDE: 'alongside', 
  HIDDEN: 'hidden', 
  OVER: 'over'
}
`

File: README.md [hunk 2]
```diff
@@ -110,23 +98,94 @@ This project is licensed under the AGPL-3.0 License - see the [LICENSE.md](LICEN
...
+
+Plugin dependencies:<br/>
+<a href="https://github.com/kaltura/playkit-js-kaltura-cuepoints">Cue Points</a><br/>
+<a href="https://github.com/kaltura/playkit-js-ui-managers">UI Managers</a>
 
 ### Dev env
-
-Node version: up to 14.18.1<br/>
+Node version: up to 14+<br/>
 If nvm installed: `nvm use` change version of current terminal to required.<br/>
```
Comment (semarche-kaltura) [inline README.md:191]: `nvm use` command requires `.nvmrc` file. Navigation plugin doesn't have such file, so OR remove the description for `nvm use` OR add missed `.nvmrc` file.

**Review by semarche-kaltura — CHANGES_REQUESTED** [review]

**Review by semarche-kaltura — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -110,23 +98,94 @@ This project is licensed under the AGPL-3.0 License - see the [LICENSE.md](LICEN
...
-Plugin configuration:<br/> > `expandOnFirstPlay`: boolean - if plugin should automatically opens on first play (default true);<br/> > `forceChaptersThumb`: boolean - force to use chapters thumbnails (default false);<br/> > `expandMode`: string - expand mode of kitchensink (AlongSideTheVideo|OverTheVideo, default "AlongSideTheVideo");<br/> > `userRole`: string - use session userId as identificator of user (anonymousRole|unmoderatedAdminRole, default "anonymousRole");<br/> > `itemsOrder`: object< string, number > - define order of Tabs (min value at the left) and Items inside group (min value at the top).<br/> itemsOrder also uses as tabs filter (all items that not included in "itemsOrder" object will be filtered out) (default:
-` { "All": 0, "Chapter": 1, "Slide": 2, "Hotspot": 3, "AnswerOnAir": 4, } ); `
+#### Configuration Structure
+
+```js
+//Default configuration
+"navigation" = {};
+//Plugin params
+"navigation" = {
+  expandOnFirstPlay?: boolean, // optional
```
Comment (semarche-kaltura) [inline README.md:114]: `position` property missed

**Review by semarche-kaltura — CHANGES_REQUESTED** [review]

**Pull request merged**